src/browser/api/external_window.ts

```diff
@@ -213,6 +213,15 @@ export function findExternalWindow(identity: Identity): Shapes.ExternalWindow |
   return externalWindows.get(uuid);
 }
 
+/*
+  Checks whether an an hwnd is a valid target for wrapping
+*/
+function doesExternalWindowExist(uuid: string): boolean {
+  const skipOpenFinWindows = false;
+  const allNativeWindows = electronApp.getAllNativeWindowInfo(skipOpenFinWindows);
+  return !!allNativeWindows.find(win => win.id === uuid);
```

Comment:
In the future, `uuid` might not mirror `id`, we should use here `getNativeWindowInfoLite` from `utils.ts` to parse these raw external window info objects